public/data/nikhil.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "nikhil",
+    "type": "personal",
+    "bio": "aspiring software engineer",
+    "avatar": "https://avatars.githubusercontent.com/u/72465503?s=400&u=d87f27cebe9055972284614285014b72bbcec3b0&v=4",
+    "links": [
+      {
+        "name": "GitHub",
+        "url": "https://github.com/nikhilxe",
+        "icon": "github"
+      },
+      {
+        "name": "Twitter",
+        "url": "https://mobile.twitter.com/Nikhilcodes0?t=Hq-B5ftunTYxJ57eDWhqvA&s=09",
```

This url will be way simple and more dynamic!! 
```suggestion
        "url": "https://twitter.com/Nikhilcodes0",
```